functions/src/weekly-markets-emails.ts

```diff
@@ -0,0 +1,69 @@
+import * as functions from 'firebase-functions'
+import * as admin from 'firebase-admin'
+
+import { Contract } from '../../common/contract'
+import { getPrivateUser, getValues, isProd, log } from './utils'
+import { filterDefined } from '../../common/util/array'
+import { sendThreeContractsEmail } from './emails'
+import { createRNG, shuffle } from '../../common/util/random'
+import { DAY_MS } from '../../common/util/time'
+
+export const weeklyMarketsEmails = functions
+  .runWith({ secrets: ['MAILGUN_KEY'] })
+  .pubsub.schedule('every 1 minutes')
```

Comment:
This is just debugging code, I assume. When do you plan to send this email? I vote at a consistent time like Monday at 12 pm ET.

Reply:
yup, just for debugging. I'll set it to that time after I merge it into main and uncomment the all users bit.